apps/src/dance/Dance.js

```diff
@@ -603,128 +604,8 @@ Dance.prototype.execute = async function() {
 
 Dance.prototype.initInterpreter = function() {
   const nativeAPI = this.nativeAPI;
-  const sprites = [];
```

Comment:
Did we need access to these sprites for some reason? I notice we lose that by this change.

Reply:
No, it's only needed for looking up the sprite given the id, which is now happening in the dance repo